src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Config.cs

```diff
@@ -38,6 +38,7 @@ public class Config
         public bool IsDNSCachingSupportedTR = false;  // this is for the tenant ring
         public string EnclaveAzureDatabaseConnString = null;
         public string UserManagedIdentityClientId = null;
+        public string MakecertPath = null;
```

Comment:
Please also update config.default.json file + Buildguide for future use.

Reply:
Updated.